theme/static/js/custom.js

```diff
@@ -1,115 +1,129 @@
-		return false;
-	});
-
-    // Restyle keywords field
-	//===================
-    var keywords = $("#keywords").text().split(",");
-    var list = $("#list-keywords");
-    for (var i = 0; i < keywords.length; i++){
-        if (keywords[i]){
-           var li = $("<li><a class='tag'></a></li>");
-            li.find('a').text(keywords[i]);
-            list.append(li);
+        if (item.find(".isUrlClickable").length) {
+            return this;    // Items already initiated
         }
-    }
-
-    $("#keywords").remove();
 
-    if ($("#citation-text").length > 0){
-        // Make links in citation clickable
-        //===================
-        var oriCitationText = $("#citation-text").text();
-        var newCitationText = oriCitationText;
-        var citationUrl;
+        var originalText = item.text().trim();
+        var newText = originalText;
+        var url;
 
         // Regular expression to find FTP, HTTP(S) and email URLs.
         var regexToken = /(((ftp|https?):\/\/)[\-\w@:%_\+.~#?,&\/\/=]+)|((mailto:)?[_.\w-]+@([\w][\w\-]+\.)+[a-zA-Z]{2,3})/g;
 
         var matchArray;
         // Extract the url
-        while( (matchArray = regexToken.exec(oriCitationText)) !== null ){
+        while ((matchArray = regexToken.exec(originalText)) !== null) {
             if (matchArray[0].slice(-1) === "." || matchArray[0].slice(-1) === ",") {
-                citationUrl = matchArray[0].slice(0, -1);
+                url = matchArray[0].slice(0, -1);
             }
             else
-                citationUrl = matchArray[0];
+                url = matchArray[0];
 
-            newCitationText = newCitationText.replace(citationUrl, "<a href=\"" + citationUrl + "\" target=\"_blank\">" + citationUrl + "</a>")
+            newText = newText.replace(url, "<span class='isUrlClickable'>" +
+                "<a href=\"" + url + "\" target=\"_blank\">" + url + "</a></span>")
         }
-        $("#citation-text").html(newCitationText);
+        $(this).html(newText);
+
+        return item;
     }
+})(jQuery);
+
+
+$(document).ready(function () {
+    // Search box toggle
+    // =================
+    $('#search-btn').on('click', function () {
+        $("#search-icon").toggleClass('fa-times margin-2');
+        $("#search-box").toggleClass('display-block animated fadeInUp');
+    });
+
+    // Smooth scrolling for UI elements page
+    // =====================================
+    $(document).ready(function () {
+        $('a[href*=#buttons],a[href*=#panels], a[href*=#info-boards], a[href*=#navs], a[href*=#alerts], a[href*=#thumbnails], a[href*=#social], a[href*=#section-header],a[href*=#page-tip], a[href*=#block-header]').bind("click", function (e) {
+            var anchor = $(this);
+            $('html, body').stop().animate({
+                scrollTop: $(anchor.attr('href')).offset().top
+            }, 1000);
+            e.preventDefault();
+        });
+        return false;
+    });
```

Comment:
I don't think so. This existed before I started working in this project. I can fix it.